dotnet/src/Connectors/Connectors.OpenAI.UnitTests/Core/OpenAIJsonSchemaTransformerTests.cs

```diff
@@ -15,12 +14,12 @@ namespace SemanticKernel.Connectors.OpenAI.UnitTests.Core;
 /// </summary>
 public sealed class OpenAIJsonSchemaTransformerTests
 {
-    private static readonly JsonSchemaMapperConfiguration s_jsonSchemaMapperConfiguration = new()
+    private static readonly AIJsonSchemaCreateOptions s_jsonSchemaCreateOptions = new()
```

Comment:
These unit test are testing implementation detail that has now been removed. I'll fix the failing build but will otherwise leave these as-is since they're testing the same configuration as the one being passed to the core client.